org.idempiere.webservices/WEB-INF/src/org/compiere/model/MWebServiceType.java

```diff
@@ -286,5 +286,45 @@ public X_WS_WebServiceFieldInput getFieldInput(String colName){
 		
 		return m_inputFieldMap.get(colName);
 	}
+
+	public static void insertParameters (Properties ctx, MWebServiceType ws, String trxName) {
+
+		if (ws.getWS_WebService().getValue().equals("ModelADService")) {
```

Comment:
java best practice:
instead of variable.equals(constant) is better to write
constant.equals(variable)

this is about safety in case the variable is null - the first version will throw an NPE the second will just return false

that applies to this line and also 295, 298, 305..309

Reply:
ok, updated